assemblies/assembly-rhdh-telemetry.adoc

```diff
@@ -8,15 +8,15 @@ The telemetry data collection feature is enabled by default. {product} sends tel
 You can disable the telemetry data collection feature based on your needs. For example, in an air-gapped environment, you can disable this feature to avoid needless outbound requests affecting the responsiveness of the {product-very-short} application. For more details, see the link:{LinkAdminGuide}#disabling-telemetry-data-collection_admin-rhdh[Disabling telemetry data collection in {product-very-short}] section.
 ====
 
-{company-name} analyzes the collected data to understand your interactions with the {product-very-short} application without disclosing your identity and compromising your privacy. {company-name} uses the collected data only for internal analysis and to improve the offerings and your experience with {product}.
+{company-name} analyzes the collected data to understand your interactions with the application and to improve the offerings and your experience with {product}.
 
 This feature collects the following data about your {product-very-short} usage:
 
 * Events of page visits and clicks on links or buttons.
 * System-related information, for example, locale, timezone, user agent including browser and OS details.
 * Page-related information, for example, title, category, extension name, URL, path, referrer, and search parameters.
 
-IP address and username are anonymized to maintain your privacy:
+IP address and username are anonymized:
```

Comment:
You could probably just remove this line, since it's pretty much repeating the next line. I'd just make this one combined bullet list.